config/base/kustomization.yaml

```diff
@@ -1,10 +1,7 @@
 resources:
 - namespace.yaml
-- crd
-- rbac/role.yaml
-- rbac/role_binding.yaml
-- rbac/leader_election_role.yaml
-- rbac/leader_election_role_binding.yaml
+- ../crd
+- ../rbac/
```

Comment:
why inconsistent trailing `/`?

Reply:
There used to be more stuff after that I deleted, and I didn't also remove the slash also, I think.